services/dummy-test-service/go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/tomdoestech/go-react-todo
```

Comment:
uuuh, this is not the way to solve the go workspace error. you're basically creating another module in this repo (each repo should only have one module at root, which we do), and this was copied over from `github.com/tomdoestech/go-react-todo`, this is the `github.com/stickianempire/stickian` repo

Reply:
Then I don't know how to solve the error. I will try to investigate more about this.